RTS/Assets/Scripts/GoTask.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GoTask : Task{
+    [SerializeField]
+    private Vector3 Destiny;
+
```

Comment:
should we change this variable to "Destination" ? paride

Reply:
done.